R/printOutput.R

```diff
@@ -4,22 +4,20 @@
 #'
 #' @param string String to print
 #' @param len1stcol length of first column
+#' @param lenCols vector with length information for all columns
+#' @param colSep string separating the columns
 #'
 #' @author Anastasis Giannousakis
 #' @export
-printOutput <- function(string, len1stcol = 67) {
+printOutput <- function(string, len1stcol = 67, lenCols = NULL, colSep = "   ") {
+
+  if (length(lenCols) > 0) len1stcol <- lenCols[1]
 
   formatstr <- function(x, len) {
```

Comment:
a bit shorter and maybe more elegant. For `is.na(x)`, I changed to return an " " string of length `len` instead of a fixed one. OK?

Reply:
OK